integreat_cms/static/src/js/analytics/hix-widget.ts

```diff
@@ -132,16 +132,20 @@ window.addEventListener("load", async () => {
         },
     } as ChartConfiguration);
 
-    const enableMT = () => {
-        const mtCheckBox = document.getElementById("id_automatic_translation") as HTMLInputElement;
-        mtCheckBox.disabled = false;
-    };
-
-    const disableMT = () => {
-        const mtCheckBox = document.getElementById("id_automatic_translation") as HTMLInputElement;
-        mtCheckBox.checked = false;
-        mtCheckBox.dispatchEvent(new Event("change"));
-        mtCheckBox.disabled = true;
+    const toggleMTCheckboxes = (disabled: boolean) => {
+        const mtForm = document.getElementById("machine-translation-form");
+        const checkboxes: NodeListOf<HTMLInputElement> = document.querySelectorAll(
+            "#machine-translation-form input[type='checkbox']"
+        );
+        checkboxes.forEach((checkbox) => {
+            if (disabled) {
+                mtForm.classList.add("pointer-events-none");
+                checkbox.classList.add("fake-disable");
+            } else {
+                mtForm.classList.remove("pointer-events-none");
```

Comment:
```suggestion
            const label = document.querySelector(`label[for='${checkbox.id}']`);
            if (disabled) {
                label.classList.add("pointer-events-none");
                checkbox.classList.add("fake-disable");
            } else {
                label.classList.remove("pointer-events-none");
```
I think we need to disable the corresponding labels as well.

Reply:
I just went for the entire surrounding div, but this is fine too, ofc.